Objects/Converters/ConverterRevit/ConverterRevitShared/ConversionUtils.cs

```diff
@@ -28,51 +28,24 @@ public partial class ConverterRevit
   {
     #region hosted elements
 
-    private bool ShouldConvertHostedElement(DB.Element element, DB.Element host)
+    // Creates an ApplicationObject with status = Skipped for each object in the `elements` property
+    // This will tell the connector to skip these elements when traversing 
+    private void SkipNestedElementsOnReceive(Base host)
```

Comment:
We should allow to pass some rules so that only elements of a specific type would be skipped.
In the case of CurtainWalls only mullions and panels, but not doors and windows.